### PR TITLE
pctl version isn't correct

### LINKED ISSUE #240: pctl version isn't correct
From the latest `v0.2.0` release:
```
$  pctl --help
NAME:
   pctl - A cli tool for interacting with profiles

USAGE:
   pctl [global options] command [command options] [arguments...]

VERSION:
   0.1.0-dev+6b75a51.2021-07-28T13:14:29Z
   ....
   
$ pctl --version
pctl version 0.1.0-dev+6b75a51.2021-07-28T13:14:29Z
```

### PR DESCRIPTION
**Pull request by Himangini**
### Description
fixed the version tag in `release.go` and updated the release section of Readme docs
fixes #240 

### Checklist
- [ ] Added tests that cover your change
- [x] Added/modified documentation as required (such as the `README.md`)
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [x] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

**Review by aclevername — DISMISSED** [review]

File: pkg/version/release.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package version
 // This file was generated by release_generate.go; DO NOT EDIT.
 
 // Version is the version number in semver format X.Y.Z
-var Version = "0.1.0"
+var Version = "0.2.1"
```
Comment (aclevername) [inline pkg/version/release.go:6]: I'm confused, this should be `0.2.0` right? since its the latest version we cut

Comment (Himangini) [reply pkg/version/release.go:6]: i thought this would go in next release since we already release 0.2.0? 🤔

Comment (aclevername) [reply pkg/version/release.go:6]: sorry I'm super tired and confused :smile: if you wanna cut a `0.2.1` PR after this is merged, then yeah this makes sense to keep in this PR :D but you also need to add a `0.2.1` release notes file too

Comment (Himangini) [reply pkg/version/release.go]: sorry I am typing in too many threads 🙈  switched it back to `0.2.0`

File: README.md [hunk 2]
```diff
@@ -379,6 +379,8 @@ Steps:
    correct by doing something like: `git log --first-parent tag1..tag2`._
 
 1. PR the release notes into main.
+_Note_ that you should also update the `var Version` in `pkg/version/release.go` file with the same tag which is created in the following step
+This should be automated with #233
```
Comment (aclevername) [inline README.md:383]:
```suggestion
1. You need to update the `var Version` in `pkg/version/release.go` file with the same tag which is created in the following step. This should be automated with #233
```

idk if this formatted correctly so double check :smile:

**Review by aclevername — APPROVED** [review]

**Pull request merged**